enhancements/node-tuning/mixed-cpu-node-plugin.md

```diff
@@ -176,51 +178,59 @@ Are the same in that context, the container has access to the same number of sha
 In other words, the device request only uses as a hint to tell the plugin that shared-cpus required for the container and
 does not indicate any actual value.
 
-The reason for specifying a device, and not, for example, an annotation,
-is because when application pod requests for a device, the scheduler makes the pod pending till device up and running.
-This gives the node plugin room for setup and not being depended on pod admission order.
-So only when the node plugin finishes the registration process with NRI and device plugin, the scheduler
-admits the application pod.
-
-Once a container under the pod requests and gets access to shared cpus, new environment variable named
-`OPENSHIFT_SHARED_CPUS=<CPU-IDs>` will be present under the container's environment variables.
-
-The mixed-cpu-node-plugin itself is essentially a pod that runs as a DaemonSet on the worker nodes.
-cluster-node-tuning-operator (NTO) will be modified to support the deployment of the DaemonSet and additional
-resources (ServiceAccount, RBAC resources, SecurityContextConstraint, etc.)
-NTO will also be responsible to watch, monitor and report the node-plugin's DaemonSet status.
-Additionally, since the plugin is part of OCP infrastructure pods, NTO annotates the pods with:
-```yaml
-target.workload.openshift.io/management: {
-  "effect":"PreferredDuringScheduling"
+The reason for specifying a device, and not an annotation,
+is to allow control on the number of workloads that can request for shared cpus.
+By populating shared cpu devices as the number of maximum workloads.
+If the number of shared cpu devices got exhausted, any new workload that requests for shared cpus device will move
+to pending state by the scheduler.
```

Comment:
> How will the maximum number of workloads be determined?

It should be specified in the Kubelet configuration file: 
https://github.com/openshift/enhancements/pull/1396/files/99d9508ac6750936842234d2d5d70f3133347346..4cfe5b9917c0335f57672bd4d1d52628894a3ea3#diff-8e686dd2a45a9ea128ad353500bcbdce821cea213decdc914bda03a09c640186R201 

Initially  we'll start with some default value.
On later stage we can expose this value through API.

Reply:
If the required value is 1, would it be better to just reject the request if it is not 1? That seems more clear to the user.

Reply:
I'm not sure what and where changes are needed for that.
@bartwensley Could please elaborate if you are familiar with those areas?

Reply:
You would do this in the new API Server Admission Hook you describe below. Instead of mutating the request if it is > 1, you would just reject the pod with a clear error so the user can correct their pod spec.

Reply:
Thank you @bartwensley. I changed according to your recommendation.